src/hooks/useSelectionLogic.ts

```diff
@@ -120,7 +120,7 @@ export function useSelectionLogic<T extends HTMLElement>({
         };
 
         // we detect move only after some small movement
-        if (calculateBoxArea(newSelectionBox) > 10) {
+        if (newSelectionBox.height + newSelectionBox.width > 8) {
```

Comment:
Thanks for creating that PR! What do you think about making it even more flexible - creating an optional parameter (like `isMinMovement` or any better name), where you would pass a method that returns `true` if minimal movement is achieved? It could be `(newSelectionBox) =>calculateBoxArea(newSelectionBox) > 10` by default, and `(newSelectionBox) => newSelectionBox.height + newSelectionBox.width > 8` in your case

Reply:
@tomaszczura I have finally gotten some time to incorporate this change into the PR. Ready for another round of review